components/layout_thread_2020/lib.rs

```diff
@@ -248,10 +260,10 @@ impl LayoutThreadFactory for LayoutThread {
         busy: Arc<AtomicBool>,
         load_webfonts_synchronously: bool,
         window_size: WindowSizeData,
-        _dump_display_list: bool,
-        _dump_display_list_json: bool,
-        _dump_style_tree: bool,
-        _dump_rule_tree: bool,
+        dump_display_list: bool,
+        dump_display_list_json: bool,
+        dump_style_tree: bool,
+        dump_rule_tree: bool,
```

Comment:
Such an unethical method. 😂 No nit here though, we don't have time for that.

Reply:
Yeah it’s pretty bad, but also copied over.